tests/Resources/ti.ui.webview.test.js

```diff
@@ -848,6 +848,48 @@ describe.androidARM64Broken('Titanium.UI.WebView', function () {
 		win.open();
 	});
 
+	it('decode url', (finish) => {
+		win = Ti.UI.createWindow({
+			backgroundColor: 'blue'
+		});
+		const webview = Ti.UI.createWebView({
+			url: 'https://test/sub/api?key=TiTeSTKEy%3D%3D&var=1234'
+		});
+
+		webview.addEventListener('load', e => {
```

Comment:
Should this ever fire a load event? The url looks invalid, so I assume it'd never load.